app/legacy_lib/to_deprecated_h.rb

```diff
@@ -13,4 +13,8 @@ def to_deprecated_h
       to_unsafe_h
     end
   end
+end
+
+# unneeded module so zeitwerk:check passes
+module ToDeprecatedH
```

Comment:
What is this?

Reply:
zeitwerk:check sees that to_deprecated_h.rb is in the autoload path so it checks that there's a constant named ToDeprecatedH in the file. If there isn't it, it errors out. This works around that.